slam-libraries/viam-cartographer/src/io/submap_painter.cc

```diff
@@ -113,5 +113,27 @@ cartographer::io::PaintSubmapSlicesResult PaintSubmapSlices(
                                                      origin);
 }
 
+// Draw a position on the current map
+void DrawPositionOnSurface(
+    cartographer::io::PaintSubmapSlicesResult* painted_slices,
+    cartographer::transform::Rigid3d global_pose, float resolution) {
+    float marker_size = .3;
+
+    auto cr = cartographer::io::MakeUniqueCairoPtr(
+        cairo_create(painted_slices->surface.get()));
+
+    // move to origin of map
+    cairo_translate(cr.get(), painted_slices->origin.x(),
```

Comment:
unsure of how this origin interacts with multiple trajectories/submaps in a pbstream. May have to test once apriori map PR goes through

Reply:
I think we should test this with apriori maps and localization before merging. Both will probably be done next week Monday.

Reply:
You tested this, right? Was moving the origin of the map an issue?

Reply:
did not appear to be an issue based off the images